data_fusion/plugins/yaml/products/layered.yaml

```diff
@@ -6,38 +6,49 @@ docstring: |
 spec:
   products:
     - name: Layered-Winds-Default
+      source_name: layered
       family: xarray_dict_to_output_format
       docstring: |
         Layered winds product using default colorbar placement.
       spec:
-        coverage_dataset: 'windspeed'
-        covg_func: masked_arrays
-        covg_args:
-          alt_varname_for_covg: wind_speed_kts
+        coverage_checker:
+          plugin:
+            name: masked_arrays
+            arguments:
+              varname: windspeed
+              alt_varname: wind_speed_kts
     - name: Layered-Winds-Two-Colorbars
+      source_name: layered
       family: xarray_dict_to_output_format
       docstring: |
         Layered winds product using default 2 colorbar placement.
 
         This example layered image includes default colorbar placement
         for both windspeed and ir products, and no colorbar for windbarbs.
       spec:
-        coverage_dataset: 'windspeed'
+        coverage_checker:
+          plugin:
+            name: masked_arrays
+            arguments:
```

Comment:
@jsolbrig I'm not sure this is doing the same thing - the "coverage_dataset" was specifying which dataset to pull the "alt_varname" variable from.  "alt_varname" is the variable to use from the "coverage_dataset" if the variable named <product> is not available.  Not sure why this is working?

Reply:
Hmmm, maybe this is working by accident. Should we take a look at this together to get it right?

Reply:
https://github.com/NRLMMD-GEOIPS/data_fusion/issues/15
I opened an issue for it.  I say we merge this and look into the variable specifications later.